.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+### Description
+
+Please include a summary of the change being made.
+
+Fixes #(issue)
+
+### Why is the change necessary?
+
+What capability does it enable? What problem does it solve?
+
+### Solution
+
+Please include an overview of the solution. Discuss trade-offs made, caveats, alternatives, etc.
+
+### Testing
+
+How was this change tested?
+
+----
+
+### Pull Request Checklist
+
+Please check all boxes (including N/A items)
+
+#### Testing
+
+- [ ] Unit tests added
+- [ ] integration test added
+
```

Comment:
May be worth adding manual testing as an item and asking why, if done, it could not be automated.

Reply:
good call, will add